package.json

```diff
@@ -42,6 +42,7 @@
     "webpack": "^5.65.0",
     "webpack-assets-manifest": "^5.0.6",
     "webpack-cli": "^4.9.1",
+    "yarn-deduplicate": "^5.0.0",
```

Comment:
I wouldn't expect this to be installed to the project, and instead should be installed as a global package on a developer's machine.

```suggestion
```